airbyte-integrations/connectors/source-pinterest/acceptance-test-config.yml

```diff
@@ -23,26 +23,24 @@ acceptance_tests:
     tests:
       - config_path: secrets/config.json
         empty_streams:
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: ad_account_analytics
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: ad_analytics
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: ad_group_analytics
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: ad_groups
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: ads
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: board_section_pins
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: board_sections
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: campaign_analytics
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: campaigns
-          - bypass_reason: The stream could return 0 records, because of low rate-limits
-            name: user_account_analytics
+          - name: conversion_tags
+            bypass_reason: Not possible to add data
+          - name: customer_lists
+            bypass_reason: Not possible to add data
+          - name: catalogs
+            bypass_reason: Not possible to add data
+          - name: catalogs_feeds
+            bypass_reason: Not possible to add data
+          - name: catalogs_product_groups
+            bypass_reason: Not possible to add data
+          - name: product_group_report
+            bypass_reason: Not possible to add data
+          - name: product_group_targeting_report
+            bypass_reason: Not possible to add data
+          - name: product_item_report
+            bypass_reason: Not possible to add data
+          - name: keyword_report
+            bypass_reason: Not possible to add data
```

Comment:
`Not possible to add data`: does it mean the stream is empty or that is has default data we can't change?
If it's completely empty should we just pass a custom `configured_catalog_path` excluding these streams?

Reply:
it is empty streams - with no data.
As far as I know "Basic Test" does not allow to provide custom catalog